selinux/qubes-meminfo-writer.te

```diff
@@ -0,0 +1,21 @@
+policy_module(qubes-meminfo-writer, 0.0.1)
+
+require {
+	type local_login_t, sysfs_t, proc_t;
+}
+
+type qubes_meminfo_writer_t;
+type qubes_meminfo_writer_exec_t;
+type qubes_meminfo_writer_var_run_t;
+init_daemon_domain(qubes_meminfo_writer_t, qubes_meminfo_writer_exec_t)
+qubes_meminfo_writer_signal(local_login_t)
+allow qubes_meminfo_writer_t self:process { fork signal_perms };
+allow qubes_meminfo_writer_t self:fifo_file rw_fifo_file_perms;
+allow qubes_meminfo_writer_t { sysfs_t proc_t }:file { open read };
+domain_use_interactive_fds(qubes_meminfo_writer_t)
```

Comment:
Why is it necessary? Is it about writing logs to stderr?

Reply:
I think so?  I don’t remember.  It could also have been about FDs passed over AF_UNIX.

Reply:
> It could also have been about FDs passed over AF_UNIX.

I don't think meminfo-writer has any.